package-managers/nuget.md

```diff
@@ -1,25 +1,76 @@
 # NuGet \(.NET\)
-
 TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-nuget)
 
-## Connecting to a NuGet feed
 
+## Connecting to a NuGet feed
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
 
+### Find your feed URL
+
+#### Public feed
+There are two types of public feeds you can connect to. A public feed you own and one from the gallery. The only real difference here, is where to find the URL of the feed.
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+
+##### Gallery feed
+When connecting to a gallery feed you first click on the feed you want to connect to and there click on the "Connect to feed" button.
+
+![Gallery feed details](assets/gallery-feed.png)
+
+A dialog opens where you will once again find your 'NuGet V3' URL.
+
+![Gallery feed URL](assets/connect-gallery-feed.png)
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+##### Private feed
+To consume private feeds you need to get consume permissions from the feed owner or admin. When you have these rights you can navigate to the 'feed details' where you can find the 'NuGet V3' URL. 
+
+![Feed details](assets/feed-details.png)
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+
+### Visual Studio
+
+#### Public feed
+If you don't know where to find your feed URL check the [Find your feed URL](#find-your-feed-url) section.
+
+#### Private feed
+
+#### Credential Provider for Visual Studio
 inspiration: http://docs.myget.org/docs/reference/credential-provider-for-visual-studio but needs to be nicer
 
-## Creating NuGet/.NET Core packages
 
+### JetBrains Rider
+
+#### Public feed
+If you don't know where to find your feed URL check the [Find your feed URL](#find-your-feed-url) section.

```

Comment:
Not sure we need to repeat these sections here. Perhaps have the find you feed URL section from earlier, and here just say "We'll assume we already have the feed's connection details at hand." - which would apply for private and public feeds. Skip the clutter, go to the meat :-)